source/jsonnet/england-wales/household/blocks/who-lives-here/anyone_else_temporarily_away_list_collector.jsonnet

```diff
@@ -68,23 +78,27 @@ local editQuestion(questionTitle) = {
     guidance: {
       contents: [
         {
-          title: 'Include people who are temporarily away',
+          title: 'Remember to Include people who are<br><br>',
```

Comment:
I can't find any better solution for adding space here. I tried a title with no list, title with an empty list and title with a list with an empty string element. None of them didn't work as expected.

Reply:
`br` is for a literal line break. We should `p` tags for this given that we don't fully support this thought schema definitions yet.

`<p>Remember to Include people who are</p>`

Applies to all occurrences.

Reply:
Changed it to paragraph, thanks!